### ISSUE TITLE
Code problem: The methods and dataset libraries seem to be missing

### ISSUE THREAD

**Issue by CLJ-3214** [mepleleo/BIA-CNNbandselection#1]
These two libraries don't seem to be open source libraries, and I haven't found them anywhere else.
I assume that this is an author's private library function, so I would like to ask the author if they can open source both libraries and specify the data set. thank you

**Comment by mepleleo** [comment]
Thank you for your interest.
(1)The data used in the paper is not my personal, but belongs to the research group, so it cannot be made public. The data are similar to publicly available hyperspectral datasets. You can experiment with public datasets or your data.
(2)from methods import train_cnn
from dataset import Pnt_eb_Dataset, lines_aug,load_data,load_data2
These two libraries are the code for training ShuffleNet V2, and I referenced this repository. 
https://github.com/NailinLiao/ModelZoo/tree/15d93f56cbe41b32e7abf8486c545518474a742a/pytorch-cifar100-master
If you are familiar with python and deep learning, this is not a problem.
If you want to run this code first, you can comment these two lines of code. Then customize load_data and load your data. This can run the dt_knn_svm() function.

**Comment by CLJ-3214** [comment]
Thank you very much for your reply. I will conduct further study and experiment.

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Jerry ***@***.***>
发送时间: 2022年5月29日 16:56
收件人: ***@***.***>
抄送: Chuanli ***@***.***>; ***@***.***>
主题: Re: [mepleleo/BIA-bandselection] Code problem: The methods and dataset libraries seem to be missing (Issue #1)


Thank you for your interest.
(1)The data used in the paper is not my personal, but belongs to the research group, so it cannot be made public. The data are similar to publicly available hyperspectral datasets. You can experiment with public datasets or your data.
(2)from methods import train_cnn
from dataset import Pnt_eb_Dataset, lines_aug,load_data,load_data2
These two libraries are the code for training ShuffleNet V2, and I referenced this repository.
https://github.com/NailinLiao/ModelZoo/tree/15d93f56cbe41b32e7abf8486c545518474a742a/pytorch-cifar100-master
If you are familiar with python and deep learning, this is not a problem.
If you want to run this code first, you can comment these two lines of code. Then customize load_data and load your data. This can run the dt_knn_svm() function.

―

**Comment by mepleleo** [comment]
————————————您好，来信已收到，谢谢。
